app/models/conference.rb

```diff
@@ -4,7 +4,9 @@ class Conference < ActiveRecord::Base
 
   has_many :attendances
   has_many :attendees, through: :attendances, source: :user
+
   validates :name, :round, presence: true
+  validate :chronology # custom validator
```

Comment:
I see what you mean. I tried to find something with more analogy to `validates :name` . Can you live with: `validate :interval` or `validate :time_span` ? If not, I go with your suggestion.